velox/functions/sparksql/Hash.cpp

```diff
@@ -325,18 +331,25 @@ class XxHash64Function final : public exec::VectorFunction {
       exec::EvalCtx& context,
       VectorPtr& resultRef) const final {
     context.ensureWritable(rows, BIGINT(), resultRef);
```

Comment:
The signature says return type is "integer", but here we use BIGINT? Looks like one of these places is wrong.

Reply:
I still wonder about this.